datadog_lambda/handler.py

```diff
@@ -7,6 +7,9 @@
 from importlib import import_module
 
 import os
+import time
+
+from datadog_lambda.tracing import emit_telemetry_on_exception_outside_of_handler
```

Comment:
Should we lazy load this in the catch? 

What's the overhead difference if we do that?

Reply:
Oh yeah, good point. What's the best way to measure the difference?

Reply:
Instrument a function with it, check the overall cold start.

Then lazy load it, build it again and instrument a function with it, check the cold start.

Compare them and see the differences.

Since you won't have the cold start tracer here, you might not be able to see the actual imports. So your best chance is to look for CloudWatch Logs to see the cold start here.

Reply:
This got refactored a bit, but I tested moving the import of `error_fallback_handler` into the catch. The traces actually do get a cold start span, and the durations look right based on the CloudWatch logs. Based on those spans ([trace from import at the top](https://ddserverless.datadoghq.com/apm/trace/14945367174940455538?colorBy=service&env=dev&graphType=flamegraph&shouldShowLegend=true&sort=time&spanID=4795421708683935480&spanViewType=metadata&timeHint=1713916422872), [trace from import in the catch](https://ddserverless.datadoghq.com/apm/trace/13634600884944720567?colorBy=service&env=dev&graphType=flamegraph&shouldShowLegend=true&sort=time&spanID=7025402365249368892&spanViewType=metadata&timeHint=1713967321993)), I'm not seeing a noticeable difference in overhead